events.html

```diff
@@ -44,6 +44,19 @@ <h3>Kitura is proud to be a part of the wider Swift community. Below you can see
     </div>
 
   <div class="past-meetups-section">
+    <div class="meet past">
+
+      <div class="meetup-image" style="background-color:rgb(234, 234, 234);"><img src="assets/calendar.png" alt="Swift Summit"></div>
+      <div class="meetup-info">
+        <h2>try! Swift 2018 - New York City, USA</h2>
+        <h3><a href="https://developer.ibm.com/swift/2018/09/01/announcing-kitura-2-5/">Blog Post: Introducing Kitura 2.5 - Ian Partridge</a></h3>
+        <div class="summary">
+          <h3>The Kitura team, represented by Chris Bailey and Ian Partridge, were on stage with the Apple SwiftNIO and Vapor teams as part of the Swift on Server Workgroup working to bring common APIs to the server! We ran a workshop demonstrating how to build and deploy a cloud-native Swift application which was very popular with the community.</h3>
```

Comment:
Ian Partridge and Chris Bailey hosted a [workshop](https://github.com/IBM/ToDoBackend), and gave a presentation on the latest updates to Kitura.  We also joined Apple and Vapor to announce a new start for the [Swift Server Work Group](https://swift.org/server/).